i18n/zh-CN/docusaurus-plugin-content-docs-community/current/how-to-vote-a-committer-pmc.md

```diff
@@ -1,10 +1,13 @@
 ---
-title: 成为InLong Committer 或 PPMC 的投票过程
+title: 成为InLong Committer 或 PMC 的投票过程
 sidebar_position: 5
 ---
 
 ## 发起社区Private邮件组投票讨论
-   任何InLong的PPMC成员都可以发起投票讨论，在PPMC发现社区贡献者任何有价值的贡献并取得候选人本人同意后，可以在InLong的private邮件列表发起讨论。讨论邮件里提议者要把候选人的贡献说清楚，并且给出复核对应贡献的地址，便于大家讨论分析。讨论邮件主送private@inlong.apache.org邮箱，讨论将持续至少72个小时，项目组成员，包括mentor们会针对提议邮件充分发表自己的看法。如下是讨论邮件样例：
+   任何InLong的PMC成员都可以发起投票讨论，PMC在发现社区贡献者任何有价值的贡献并取得候选人本人同意后，可以在InLong的private邮件列表发起讨论。
```

Comment:
Suggest adding one blank between English and Chinese characters.

Reply:
Ok, let me fix it

Reply:
@healchow, please review again!